core/src/main/java/com/googlecode/psiprobe/tools/SecurityUtils.java

```diff
@@ -29,11 +29,6 @@
 public class SecurityUtils {
 
   /**
-   * Instantiates a new security utils.
-   */
-  private SecurityUtils() {}
```

Comment:
PR is coming with this fix.  I did review my JavaBeanTester code to see what was otherwise causing failure.  It was not internals for testing no-arg constructor.  It was actually kicking off a bug.  So I filed an issue to self there and will ignore related test case.